ntbs-service/deployments/live.yml

```diff
@@ -47,8 +47,10 @@ spec:
                 key: migration
           - name: AdfsOptions__Wtrealm
             value: # ???
+          - name: AdfsOptions__AdfsUrl
+            value: "https://fs.softwire.com"
           - name: AdfsOptions__AdGroupsPrefix
-            value: ""
+            value: "pheNtbs - "
```

Comment:
Don't think we want these changes in the `live.yml` file - this one is being prepped for PHE release, which will definitely not be on Softwire's AD ;)